esrally/driver/runner.py

```diff
@@ -1027,7 +1029,19 @@ class CreateMlDatafeed(Runner):
     def __call__(self, es, params):
         datafeed_id = mandatory(params, "datafeed-id", self)
         body = mandatory(params, "body", self)
-        es.xpack.ml.put_datafeed(datafeed_id=datafeed_id, body=body)
+        try:
+            es.xpack.ml.put_datafeed(datafeed_id=datafeed_id, body=body)
+        except elasticsearch.TransportError as e:
+            # fallback to old path
+            if e.status_code == 400:
+                es.transport.perform_request(
+                    "PUT",
+                    "/_xpack/ml/datafeeds/" + datafeed_id,
```

Comment:
What if the user specifies a [datafeed-id](https://esrally.readthedocs.io/en/stable/track.html#create-ml-datafeed) as integer? I was wondering where we should stay consistent with similar code in hot paths and use `"/_xpack/ml/datafeeds/%s" % datafeed_id,` instead.